src/index.coffee

```diff
@@ -173,7 +178,7 @@ module.exports.configure = (options, app, callback) ->
 
     api.setupModels()
 
-    try {requestsToSave, models} = getRequests options.root
+    try {requestsToSave, models} = getRequests options.modelsPath
```

Comment:
No `options` is given in paramater. It's only built if it's given as a string or if the root parameter is not given.